lib/core/src/Cardano/DB/Sqlite.hs

```diff
@@ -206,35 +207,63 @@ destroyDBLayer (SqliteContext {getSqlBackend, trace, dbFile}) = do
                            Internal / Database Setup
 -------------------------------------------------------------------------------}
 
+-- | Represents an automatic migration to be performed on a database.
+data AutoMigration
+    = AutoMigrationNone
+      -- ^ Do not perform an automatic migration.
```

Comment:
This case seems unused? I can see the conceptual reason to have it, but in the meantime, why bother?

Reply:
@KtorZ wrote:
> This case seems unused? I can see the conceptual reason to have it, but in the meantime, why bother?

It has actually been rather useful during testing to be able to temporarily change the migration type to `AutoMigrationNone`. Nevertheless, it's easy to add back in if we ever need it, so I have no objection to removing it.

**Fixed, see:** https://github.com/input-output-hk/cardano-wallet/commit/6f9bddb18f01826162dc924251cf64e90e71391e